### PR TITLE
Add editor prefab and also spawn bot for 1P

### PR DESCRIPTION
**Pull request by khooroko**
**Description**
Just for developer convenience. Also will now spawn the "bot" when going into singleplayer from lobby.
Also added few more scripts to the "localScripts" in the OVR prefab so that its camera can be turned off via script

@lyhvictoria

**Impact**
- [x] Medium

**Test Plan**
Try playing from both Quest and editor, should work

### CODE REVIEW

File: Assets/Scripts/ToolHuman.cs [hunk 1]
```diff
@@ -9,11 +9,15 @@ public class ToolHuman : Tool {
     // Start is called before the first frame update
     protected override void Start() {
         base.Start();
-        hapticFeedback = GetComponentInParent<HapticFeedback>();
+        if (!Application.isEditor) {
```
Comment (lyhvictoria) [inline Assets/Scripts/ToolHuman.cs]: I'm not sure but would it be better to check this `Application.isEditor` once and save it to a bool than call it every time?

**Review by lyhvictoria — APPROVED** [review]
Works fine 👍

File: Assets/Scripts/ToolHuman.cs (hunk 1 above)
Comment (khooroko) [reply Assets/Scripts/ToolHuman.cs]: not sure either but a very valid suggestion

Comment (lyhvictoria) [reply Assets/Scripts/ToolHuman.cs]: for now since there's only 2 it should be fine. You can merge first.

Comment (khooroko) [reply Assets/Scripts/ToolHuman.cs]: there's actually at least 6 so i changed it to also be consistent with handling oculus check. tested to work

**Pull request merged**